rdrf/rdrf/migrations/0040_mongo_django.py

```diff
@@ -9,7 +9,9 @@
     mongo_django = undjango_mongo = None
 
 def copy_collections(apps, schema_editor):
+    print("in copy_colections")
```

Comment:
Do you want to keep this debug code in the migrations?

Reply:
No, I over looked this, I'll remove.